dev-dotnet/dotnet-reference-packages/files/dotnet-reference-packages-fix-bashisms.patch

```diff
@@ -0,0 +1,13 @@
+Index: source-build-reference-packages-bcdc58a2ae9c2c06d769acfb8375d27adf5b2332/src/targetPacks/assemble.targets
+===================================================================
+--- source-build-reference-packages-bcdc58a2ae9c2c06d769acfb8375d27adf5b2332.orig/src/targetPacks/assemble.targets
++++ source-build-reference-packages-bcdc58a2ae9c2c06d769acfb8375d27adf5b2332/src/targetPacks/assemble.targets
```

Comment:
Consider cleaning this up, per [these guidelines](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto). It might look like:

```suggestion
# Add a helpful comment here (this is optional though)
--- a/src/targetPacks/assemble.targets
+++ b/src/targetPacks/assemble.targets
```

Reply:
Same reply as here: https://github.com/gentoo/gentoo/pull/21112#discussion_r645053761
I'd rather not go into each patch and edit it manually if I don't have to (except for the header), though I might be able to configure quilt to get rid of that index line.

Reply:
Disregard. Apparently this can be fixed by setting `QUILT_REFRESH_ARGS="--no-index --no-timestamp -p ab"` in ~/.quiltrc